code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -1160,3 +1160,29 @@
 /datum/species/golem/mhydrogen/on_species_loss(mob/living/carbon/C)
 	REMOVE_TRAIT(C, TRAIT_ANTIMAGIC, SPECIES_TRAIT)
 	return ..()
+
+/datum/species/golem/pizza

```

Comment:
ok

Reply:
wait what, i have that on a different branch